docs/installation/web-interface.md

```diff
@@ -12,10 +12,15 @@ nav_order: 2
 
    ```console
    $ mkdir -p workspace && cd workspace
-   $ docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest
+   $ docker run -p 8080:8080 -v "${PWD}:/workspace" -v /var/run/docker.sock:/var/run/docker.sock --rm -it quay.io/konveyor/move2kube-aio:latest
    ```
    Access the UI in `http://localhost:8080/`.
 
+   > Note: There is a known issue when using WSL with the above command.  
+   The CNB containerization option will not be availabe.  
+   Also empty folders may be created in the root directory.  
```

Comment:
Let's be specific and say a "/workspace" empty directory will be created.

Reply:
done